lms/djangoapps/grades/tests/test_new.py

```diff
@@ -239,56 +240,155 @@ def test_save_and_load(self):
 
 
 @ddt.ddt
-class TestMultipleProblemTypesSubsectionScores(ModuleStoreTestCase, ProblemSubmissionTestMixin):
+class TestMultipleProblemTypesSubsectionScores(SharedModuleStoreTestCase):
```

Comment:
Per [response](https://github.com/edx/edx-platform/pull/13626/commits/969c75a7d17eabef2867dc0b069932869a793544) to @adampalay's question on our earlier fix for mis-implementations of RandomizedContentBlock and DragNDrop, the expectation is that this test will provide coverage for those 2 blocks not implementing `max_score` when they should. 

Have you verified that your test fails if you remove our earlier [fix](https://github.com/edx/edx-platform/pull/13626/commits/969c75a7d17eabef2867dc0b069932869a793544)?